pkg/systemd/overview-cards/uncleanShutdownStatus.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { Button } from "@patternfly/react-core/dist/esm/components/Button/index.js";
+import { Flex, FlexItem } from "@patternfly/react-core/dist/esm/layouts/Flex/index.js";
+import { ExclamationTriangleIcon} from "@patternfly/react-icons";
+
+import cockpit from "cockpit";
+
+import './uncleanShutdownStatus.scss';
+
+const _ = cockpit.gettext;
+
+export const UncleanShutdownStatus = () => {
+    const [uncleanShutdownId, setUncleanShutdownId] = useState(() => {
+        cockpit.spawn(
+            ["last", "-xn2", "shutdown", "reboot"],
+            { err: "ignore", }
+        ).then((data) => {
+            const shutdownLines = data
+                .split("\n")
+                .filter((line) => line.startsWith("shutdown"));
```

Comment:
Looking at this implementation, this doesn't seem correct:

```
[jelle@carbon][~/projects/cockpit/unclean-shutdown]%last -xn2 shutdown reboot
reboot   system boot  6.16.4-arch1-1   Mon Sep  1 08:45   still running
shutdown system down  6.16.4-arch1-1   Sun Aug 31 20:14 - 08:45  (12:30)
```

That shutdown was clean

Reply:
Oh right `shutdown` is filtered but then still:
```
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 11:46   still running
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 11:30 - 11:45  (00:15)
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 11:26 - crash  (00:03)
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 11:24 - crash  (00:02)
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 11:17 - crash  (00:07)
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 10:06 - 10:07  (00:00)
reboot   system boot  6.15.10-200.fc42 Tue Aug 26 03:59 - 03:59  (00:00)
reboot   system boot  6.14.0-63.fc42.x Tue Aug 26 03:50 - 03:59  (00:09)
```

Some of those are valid sessions.

Reply:
Running a VM and doing a  "Shutdown force off" gives me:

```
[root@fedora-42-127-0-0-2-2201 ~]# last -xn8 reboot shutdown
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 12:08   still running
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 12:06   still running
```

The first line is a still running session:

```
[root@fedora-42-127-0-0-2-2201 ~]# last
root     pts/0        172.27.0.2       Mon Sep  1 12:08   still logged in
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 12:08   still running
reboot   system boot  6.15.10-200.fc42 Mon Sep  1 12:06   still running
```